### PR TITLE
Provide simplified way to get directly to sources/targets/move() without Session

### PR DESCRIPTION
**Pull request by vinceatbluelabs**
Now that we have a Session class which provides configurability without having to think too much about how the different objects handle their dependencies, I'd like to make things even simpler for folks with even simpler use cases, by giving the ability to skip grabbing the Session object at all, and don't want to figure out db-facts.

Example:

```python
#!/usr/bin/env python3

# Pull in the records-mover library - be sure to run the pip install above first!
from records_mover import sources, targets, move
from pandas import DataFrame
import sqlalchemy
import os

sqlalchemy_url = f"postgresql+psycopg2://username:{os.environ['DB_PASSWORD']}@hostname/database_name"
db_engine = sqlalchemy.create_engine(sqlalchemy_url)

df = DataFrame.from_dict([{'a': 1}]) # or make your own!

source = sources.dataframe(df=df)
target = targets.table(schema_name='myschema',
                       table_name='mytable',
                       db_engine=db_engine)
results = move(source, target)
```

This compares with the version here:

```python
#!/usr/bin/env python3

# Pull in the records-mover library - be sure to run the pip install above first!
from records_mover import Session
from pandas import DataFrame
import sqlalchemy

session = Session()
records = session.records

db_engine = session.get_default_db_engine()

df = DataFrame.from_dict([{'a': 1}]) # or make your own!

source = records.sources.dataframe(df=df)
target = records.targets.table(schema_name='myschema',
                               table_name='mytable',
                               db_engine=db_engine)
results = records.move(source, target)

df = DataFrame.from_dict([{'a': 1}]) # or make your own!

source = sources.dataframe(df=df)
target = targets.table(schema_name='myschema',
                       table_name='mytable',
                       db_engine=db_engine)
results = move(source, target)
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,121 @@
...
+
+Installing:
+
+```sh
+pip3 install 'records_mover[cli,postgres,redshift]'
+```
+
+Loading a CSV into a database:
+
+TODO
```
Comment (bluelabsbutler) [inline README.md:39]: Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker

File: README.md [hunk 2]
```diff
@@ -1,76 +1,123 @@
...
-The record mover can be exended expand to handle additional database
-and data file types by building on top of their
-[SQLAlchemy](https://www.sqlalchemy.org/) drivers, and is able to
-auto-negotiate the most efficient way of moving data from one to the
-other.
+Records mover can be exended expand to handle additional databases
+and data file types.  Databases are supported by building on top of
+their [SQLAlchemy](https://www.sqlalchemy.org/) drivers.  Records
+mover is able to auto-negotiate the most efficient way of moving data
+from one to the other.
```
Comment (vinceatbluelabs) [inline README.md:27]: I did a little general cleanup on the README along the way.

File: README.md [hunk 3]
```diff
@@ -1,76 +1,123 @@
...
+
+```sh
+pip3 install 'records_mover[cli,postgres,redshift]'
+```
+
+Loading a CSV into a database:
+
+```sh
+mvrec file2table foo.csv redshiftdb1 myschema1 mytable1
+```
```
Comment (vinceatbluelabs) [inline README.md:41]: As loading CSVs is one of the more common needs folks seem to have, I wanted to make sure to add an example for it, in addition to the more advanced copy-a-table-between-databases case.

File: README.md [hunk 4]
```diff
@@ -1,76 +1,123 @@
...
+db_engine = sqlalchemy.create_engine(sqlalchemy_url)
+
+df = DataFrame.from_dict([{'a': 1}]) # or make your own!
+
+source = sources.dataframe(df=df)
+target = targets.table(schema_name='myschema',
+                       table_name='mytable',
+                       db_engine=db_engine)
+results = move(source, target)
+```
```
Comment (vinceatbluelabs) [inline README.md:90]: This is an example of using the simplified usage, where we're not using a Session object at all.

File: README.md [hunk 5]
```diff
@@ -80,8 +127,12 @@ target = records.targets.table(schema_name='myschema',
...
-[here](./records_mover/records/sources/factory.py), and the
-targets supported can be found [here](./records_mover/records/targets/factory.py).
+df = DataFrame.from_dict([{'a': 1}]) # or make your own!
+
+source = sources.dataframe(df=df)
+target = targets.table(schema_name='myschema',
+                       table_name='mytable',
+                       db_engine=db_engine)
+results = move(source, target)
+```
```
Comment (vinceatbluelabs) [inline README.md:138]: And this is the more elaborate usage where we have a Session.

**Pull request merged**